keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2025,6 +2006,132 @@ Beta:
   - Automatically manages versions of mandatory/suggested configuration
   - Discrimates paginated LIST requests
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: APIPriorityAndFairness
+    - Components depending on the feature gate:
+      - kube-apiserver
+
+* **Does enabling the feature change any default behavior?** Yes, requests that
+  weren't rejected before could get rejected while requests that were rejected
+  previously may be allowed. Performance of kube-apiserver under heavy load
+  will likely be different too.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?** Yes.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  The feature will be restored.
+
+* **Are there any tests for feature enablement/disablement?** No. Manual tests
+  will be run before switching feature gate to beta.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?** A
+  misconfiguration could cause apiserver requests to be rejected, which could
+  have widespread impact such as: (1) locking an administrator out of their
```

Comment:
Is (1) really possible?  I thought requests from users in the `system:admins` group are unregulated by this feature.

Reply:
I think the question is wider than "can we misconfigure this?". Given how all apiserver requests will pass through flowcontrol, bugs could certainly make flowcontrol reject administrator requests.

Also, this is mentioned as a potential failure earlier in the KEP.

Reply:
We believe a "misconfiguration" cannot cause this. A bug could, but that's not what the text says :)

Reply:
Ack, done.